common/src/model_deployment/deploy.py

```diff
@@ -296,7 +319,7 @@ def main(args):
         endpoint_name=endpoint_name,
         deployment_name=deployment_name,
         model_id=model_id,
-        args=args
+        args=args,
```

Comment:
Please log all the args.

Reply:
Yes, generating those in other seperate PR which includes logging details for all the common component
i.e registration, deployment and local_validation